server/cache.go

```diff
@@ -316,6 +316,10 @@ type clusterInfo struct {
 	meta    *metapb.Cluster
 	stores  *storesInfo
 	regions *regionsInfo
+
+	// For test cases.
+	fakeLeaderCount map[uint64]int // storeID -> leader count.
```

Comment:
Hi @coocood 

Do you have any better way to mock the leader and region count in test?